apps/camera/style/recording-timer.css

```diff
@@ -5,6 +5,7 @@
   left: 0;
   z-index: 1;
 
+  visibility: hidden;
```

Comment:
So you also need to make sure that visibility works well with opacity transition. See https://github.com/mozilla-b2g/gaia/blob/master/apps/camera/style/hud.css for an example.